core/domain/collection_domain_test.py

```diff
@@ -221,42 +231,43 @@ def test_collection_node_exploration_id_validation(self):
         collection_node0.exploration_id = 2
         self._assert_validation_error('Expected exploration ID to be a string')
 
-    def test_collection_node_prerequisite_skills_validation(self):
+    def test_collection_node_prerequisite_skill_ids_validation(self):
         collection_node0 = self.collection.get_node('exp_id_0')
 
-        collection_node0.prerequisite_skills = {}
+        collection_node0.prerequisite_skill_ids = {}
         self._assert_validation_error(
-            'Expected prerequisite_skills to be a list')
+            'Expected prerequisite_skill_ids to be a list')
 
-        collection_node0.prerequisite_skills = ['skill0a', 'skill0a']
+        collection_node0.prerequisite_skill_ids = ['skill0a', 'skill0a']
         self._assert_validation_error(
-            'The prerequisite_skills list has duplicate entries')
+            'The prerequisite_skill_ids list has duplicate entries')
 
-        collection_node0.prerequisite_skills = ['skill0a', 2]
+        collection_node0.prerequisite_skill_ids = ['skill0a', 2]
         self._assert_validation_error(
             'Expected all prerequisite skills to be strings')
 
-    def test_collection_node_acquired_skills_validation(self):
+    def test_collection_node_acquired_skill_ids_validation(self):
         collection_node0 = self.collection.get_node('exp_id_0')
 
-        collection_node0.acquired_skills = {}
-        self._assert_validation_error('Expected acquired_skills to be a list')
+        collection_node0.acquired_skill_ids = {}
+        self._assert_validation_error(
+            'Expected acquired_skill_ids to be a list')
 
-        collection_node0.acquired_skills = ['skill0a', 'skill0a']
+        collection_node0.acquired_skill_ids = ['skill0a', 'skill0a']
         self._assert_validation_error(
-            'The acquired_skills list has duplicate entries')
+            'The acquired_skill_ids list has duplicate entries')
 
-        collection_node0.acquired_skills = ['skill0a', 2]
+        collection_node0.acquired_skill_ids = ['skill0a', 2]
         self._assert_validation_error(
             'Expected all acquired skills to be strings')
```

Comment:
Should this error message be updated to represent skill IDs instead? Ditto for any other error messages that might refer to skills when skill IDs are actually meant.